features/Support/search_page.rb

```diff
@@ -0,0 +1,26 @@
+require 'page-object'
+
+
+class SearchThePage
+	include PageObject
+
+	page_url "http://cscc.edu"
+	text_field(:id, "input")
```

Comment:
You've named this element `id` but haven't given it an identifier. Should be something like:

```ruby
text_field(:search, :id => "input")
```